src/clr-angular/popover/signpost/signpost.spec.ts

```diff
@@ -97,6 +103,50 @@ export default function(): void {
         expect(this.ifOpenService.open).toBe(false);
       });
     });
+
+    describe('aria-control values', () => {
+      spec(ClrSignpost, TestDefaultSignpost, ClrSignpostModule);
+
+      function checkAriaControlsId(id: string, element: HTMLElement) {
+        const triggerControlsValue = element.querySelector('.signpost-action').getAttribute('aria-controls');
+        const closeControlsValue = element
+          .querySelector('.signpost-content > .signpost-wrap >' + ' .signpost-content-header > .signpost-action')
```

Comment:
Could this querySelector be simplified? Seems like it doesn't need to append two separate strings.

Reply:
I simplified this.